docker/fedora/keylime_rust.Dockerfile

```diff
@@ -62,3 +64,10 @@ WORKDIR ${KEYLIME_HOME}
 RUN git clone https://github.com/keylime/keylime.git && \
 cd keylime && \
 cp keylime.conf /etc/keylime.conf
+
+# clone and build rust-keylime
+WORKDIR ${RUST_KEYLIME_HOME}
+RUN git clone https://github.com/keylime/rust-keylime.git && \
+cd rust-keylime && \
+make && \
+make install
```

Comment:
These changes will result in a huge layer in the resulting container as all intermediary compilation artifacts would be included in it (I mean many GB). Consider removing the artifacts after the installation (e.g. running `cargo clean`)